crates/polars-python/src/lazyframe/visitor/expr_nodes.rs

```diff
@@ -793,8 +808,16 @@ pub(crate) fn into_py(py: Python<'_>, expr: &AExpr) -> PyResult<PyObject> {
                 FunctionExpr::BinaryExpr(_) => {
                     return Err(PyNotImplementedError::new_err("binary expr"))
                 },
-                FunctionExpr::Categorical(_) => {
-                    return Err(PyNotImplementedError::new_err("categorical expr"))
+                FunctionExpr::Categorical(catfun) => match catfun {
```

Comment:
This can remain NotImplemented.

Reply:
Reverted the whole file, https://github.com/pola-rs/polars/pull/20211/commits/0a83dbef9314ff854a76a6aafb2ed65ea88dcaeb

Reply:
Yes. :+1: